core/data/path.go

```diff
@@ -41,6 +42,15 @@ func GetAttrPath(inAttrName string) (attrName string, attrPath string, pathType
 				attrPath = inAttrName[idx+2:]
 			}
 		}
+	} else if inAttrName[0] == '$' && inAttrName[1] == '{' {
```

Comment:
We will be adding ${activity.*} and ${trigger.*} and so on. Hence, decided to go with this approach.